versioned_docs/version-2.11.x/security-tls-authentication.md

```diff
@@ -241,14 +286,4 @@ For example:
 
 Configure `tlsTrustStorePath` when you set `useKeyStoreTls` to `true`.
 
-:::
```

Comment:
Don't remove this.

Reply:
It's not removed but duplicate. The content is introduced in [another topic](https://github.com/apache/pulsar-site/pull/432/files#diff-830b436040ec2c3350121844a43740266ff43ebff3f949287c1812632b7b0e63L576).

Reply:
Ok.